doc/Data Plane Code Generation/Subclients.md

```diff
@@ -0,0 +1,233 @@
+# Customizing the generated code
+
+## Subclients
```

Comment:
It would be good to give guidance on how a partner could create multiple service clients as well.  Is the guidance for this just to put them in separate operation groups?